docs/content/docs/getting-started/use-built-provider.md

```diff
@@ -11,75 +11,198 @@ weight: 40
+Finally, save the file.
+
 {{< /details >}}
 
-{{< details "Other: substitute your architecture" >}}
+{{< details "Windows (Vista and above)" >}}
+
+First, you need to find the location where built provider binaries are created. To do this, run this command and make a note of the path value:
+
 ```bash
-mkdir -p ~/.terraform.d/plugins/registry.terraform.io/hashicorp/google/9.0.0/myarch_amd64
-mkdir -p ~/.terraform.d/plugins/registry.terraform.io/hashicorp/google-beta/9.0.0/myarch_amd64
-ln -s $GOPATH/bin/terraform-provider-google ~/.terraform.d/plugins/registry.terraform.io/hashicorp/google/9.0.0/myarch_amd64/terraform-provider-google_v9.0.0
-ln -s $GOPATH/bin/terraform-provider-google-beta ~/.terraform.d/plugins/registry.terraform.io/hashicorp/google-beta/9.0.0/myarch_amd64/terraform-provider-google-beta_v9.0.0
+echo %GOPATH%
+```
+
+Next, create an empty configuration file. The location does not matter and could be in your home directory or a specific project directory. The extension `.tfrc` is required but the file name can be whatever you choose.
+
+If you are unsure where to put the file, put it in the `%APPDATA%` directory (use `$env:APPDATA` in PowerShell to find its location on your system).
+
+```powershell
+# create an empty file
+type nul > "$($env:APPDATA)\tf-dev-override.tfrc"
+
+# open the file with a text editor of your choice, e.g:
+notepad "$($env:APPDATA)\tf-dev-override.tfrc"
+```
+
+Open the empty file with a text editor and paste in these contents:
+
+```hcl
+provider_installation {
+
+  # Developer overrides will stop Terraform from downloading the listed
+  # providers their origin provider registries.
+  dev_overrides {
+      "hashicorp/google" = "<REPLACE-ME>\bin"
+      "hashicorp/google-beta" = "<REPLACE-ME>\bin"
+  }
+
+  # For all other providers, install them directly from their origin provider
+  # registries as normal. If you omit this, Terraform will _only_ use
+  # the dev_overrides block, and so no other providers will be available.
+  direct {}
+}
 ```
+
+Edit the file to replace `<REPLACE-ME>` with the output you saved from the first `echo` command, making sure to keep `\bin` at the end of the path. The full path is required and environment variables cannot be used. For example, `C:\Users\MyUserName\go\bin` is a valid path for a user called `MyUserName`.
+
+Finally, save the file.
+
 {{< /details >}}
 
-Once this setup is complete, terraform will automatically use the binaries generated by the `make provider` commands in the `terraform-provider-google` and `terraform-provider-google-beta` repositories instead of downloading the latest versions.
+This CLI configuration file you created in the steps above will allow Terraform to automatically use the binaries generated by the `make build` commands in the `terraform-provider-google` and `terraform-provider-google-beta` repositories instead of downloading the latest versions. All other providers will continue to be downloaded from the public Registry as normal.
```

Comment:
```suggestion
This CLI configuration file you created in the steps above will allow Terraform to automatically use the binaries generated by the `make provider` commands in the `terraform-provider-google` and `terraform-provider-google-beta` repositories instead of downloading the latest versions. All other providers will continue to be downloaded from the public Registry as normal.
```